pop-api/src/v0/README.md

```diff
@@ -0,0 +1,18 @@
+# Pop API V0
+
+The version `0` of Pop API features integrations with the following pallets:
+- [Fungibles](#fungibles)
+
+
+---
+### FUNGIBLES
+```rust
+#[runtime::pallet_index(150)]
+pub type Fungibles = fungibles::Pallet<Runtime>;
```

Comment:
I am not sure this is necessary to have here. But it seemed like including the index of this pallet was interesting info.